modules/exploits/windows/browser/malwarebytes_update_exec.rb

```diff
@@ -35,7 +35,7 @@ def initialize(info = {})
       'References'     =>
         [
           [ 'CVE', '2014-4936' ],
-          [' OSVDB', '116050'],
+          ['OSVDB', '116050'],
```

Comment:
:thinking:

Reply:
I'd also note this doesn't follow the pre and post spacing that the above entry has, so would be good to follow that and keep things consistent within the same file.

Reply:
Fixed in a075c676a6caf7f02a806d7f3adb314fa8130c88